#  Channel and PWM settings for the throttle actuator.
lat_lon_command_topic: '/lat_lon_joystick/command'
throttle_actuator:
  channel:    0
  max_pulse:  500
  zero_pulse: 370
  min_pulse:  220

# Channel and PWM settings for the steering actuator.
steering_actuator:
  channel:     1
  max_pulse:  460 # left
  zero_pulse: 360
  min_pulse:  260 # right
